climada/hazard/tc_tracks.py

```diff
@@ -460,7 +552,7 @@ def read_ibtracs_netcdf(self, provider=None, storm_id=None,
                 'name': track_ds.name.astype(str).item(),
                 'sid': track_ds.sid.astype(str).item(),
                 'orig_event_flag': True,
-                'data_provider': provider,
+                'data_provider': provider_str,
                 'basin': track_ds.basin.values[0].astype(str).item(),
                 'id_no': track_ds.id_no.item(),
                 'category': category[i_track],
```

Comment:
Should we somehow define a constant that maps the IBTrACS names to climada TCTracks names? I'm thinking something like
```
IBTRACS_CLIMADA_VAR_NAME = {
    'radius_max_wind': 'rmw',
    'radius_oci': 'roci',
    'max_sustained_wind': 'wind',
    'central_pressure': 'pres',
    'environmental_pressure': 'poci'
}
```
And then use that to fill in those variables here. Or would this be more confusing because `wind` and `pres` depend on the provider and hence it's not a direct mapping? Just an idea, but I leave that up to you.

Reply:
This comment refers to lines 538-548, sorry it's been misplaced, so in terms of create the single-track xr.Dataset.

Reply:
I don't think there is much value to this - just another constant that is only used in a single place, right?